steamapi.cpp

```diff
@@ -1,6 +1,5 @@
-#include <string.h>
-#include <malloc.h>
 #include <stdlib.h>
+#include <string.h>
```

Comment:
Should be 
```cpp
#include <cstdlib>
#include <cstring>
```

Reply:
I don't have any intention of updating the other headers. I just reordered them. I will fix it so only malloc.h is removed.

Reply:
You will not only remove `malloc.h`, you will add `string.h`.

Reply:
I have changed the PR.